geoapps/inversion/driver.py

```diff
@@ -294,60 +296,70 @@ def get_regularization(self):
 
             reg_p = regularization.Sparse(
                 self.inversion_mesh.mesh,
-                indActive=self.models.active_cells,
+                active_cells=self.models.active_cells,
                 mapping=wires.p,  # pylint: disable=no-member
-                gradientType=self.params.gradient_type,
+                gradient_type=self.params.gradient_type,
                 alpha_s=self.params.alpha_s,
-                alpha_x=self.params.alpha_x,
-                alpha_y=self.params.alpha_y,
-                alpha_z=self.params.alpha_z,
+                length_scale_x=self.params.alpha_x,
```

Comment:
maybe we should follow suit with the rename?  more descriptive

Reply:
Yup, done.